Use `flutter pub add` for dependencies in networking cookbooks (#8729)
flutter/website: src/cookbook/networking/background-parsing.md

The `http` package makes it easier to perform network
requests, such as fetching data from a JSON endpoint.

> To install the `http` package,
> add it to the dependencies section of the `pubspec.yaml` file
> by running `flutter pub add`:

Reviewer: **suggestion:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
To install the `http` package, run `flutter pub add`.
This command adds `http` to the `dependencies` section of the `pubspec.yaml` file:
```
